common/xml/fpga_architecture.xsd

```diff
@@ -293,28 +293,16 @@
     </xs:restriction>
   </xs:simpleType>
 
-  <xs:simpleType name="pb_io_equivalent">

```

Comment:
I think this property is still valid, just not where we currently have it?
http://vtr-verilog-to-routing.readthedocs.io/en/latest/arch/reference.html#tag-pb-type-input

Reply:
Updated